x-pack/plugins/monitoring_collection/README.md

```diff
@@ -2,4 +2,45 @@
 
 ## Plugin
 
-This plugin allows for other plugins to add data to Kibana stack monitoring documents.
\ No newline at end of file
+This plugin allows for other plugins to add data to Kibana stack monitoring documents.
+
+## OpenTelemetry Metrics
+
+TODO: explain how to instrument the code with `@opentelemetry/api-metrics` so that the steps below will work with metrics
```

Comment:
We'll need to do this to review the PR, so might be good to include something simple like a plugin setup counter.

Reply:
yeah. I've added a very simple example to it. let me know if it's clear enough